privacy_evaluator/models/tf/cnn.py

```diff
@@ -122,5 +149,6 @@ def __init__(
     def call(self, x):
         out = self.conv1(x)
         out = self.conv2(out)
+        out = self.conv3(out)
         out = self.fc(out)
         return out
```

Comment:
Please add documentation to this file

Reply:
The last editor can have a look at here, and maybe you just pull the team2sprint3 branch and do the similar.